part2/stages/Remove-partitions

```diff
@@ -0,0 +1,29 @@
+#!/bin/ash
+#
+# Copyright (c) 2017 Assured Information Security, Inc.
+# 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+# 
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+
+. ${SCRIPT_DIR}/functions
+
+. ${DISK_CONF}
+# ^^^ defines ${TARGET_DISK}
+
+do_cmd dd if=/dev/zero of=/dev/${TARGET_DISK} bs=512 count=1
```

Comment:
The output of this (and potentially the next commands) gets printed in the main terminal, then quickly gets cleared out by the next dialog.
We try not to do this, please redirect stdout to a more appropriate location (or /dev/null).